source/_integrations/home_connect.markdown

```diff
@@ -682,6 +682,7 @@ Both entities can use these options, but the availability of these will depend o
     - **Off**: `laundry_care_washer_enum_type_spin_speed_off`
     - **400 rpm**: `laundry_care_washer_enum_type_spin_speed_r_p_m400`
     - **600 rpm**: `laundry_care_washer_enum_type_spin_speed_r_p_m600`
+    - **700 rpm**: `laundry_care_washer_enum_type_spin_speed_r_p_m700`
```

Comment:
Is it `r_p_m700` or `r_p_m_700`?

Reply:
Is it `r_p_m_700`, but to be homogeneous with the rest of the documentation, I leave it as `r_p_m700`. I will fix it after the merge conflict that will result from merge of #38130